libcxx/include/__exception/exception_ptr.h

```diff
@@ -64,6 +64,9 @@ class _LIBCPP_EXPORTED_FROM_ABI exception_ptr {
   friend _LIBCPP_HIDE_FROM_ABI exception_ptr make_exception_ptr(_Ep) _NOEXCEPT;
 
 public:
+  // exception_ptr is basically a COW string.
```

Comment:
Let's add a release note for this optimization (for all the types, not just exception_ptr). No need to list them all, just mention we now recognize a bunch of types as trivially relocatable.